LootLockerServerSDK/Source/LootLockerServerSDK/Private/ServerAPI/LootLockerServerMetadataRequest.cpp

```diff
@@ -222,3 +366,65 @@ void ULootLockerServerMetadataRequest::GetMetadata(const ELootLockerServerMetada
 		OnComplete.ExecuteIfBound(SingleEntryResponse);
 	}));
 }
+
+void ULootLockerServerMetadataRequest::SetMetadata(const ELootLockerServerMetadataSources Source, const FString& SourceID, const TArray<FLootLockerServerSetMetadataAction>& MetadataToActionsToPerform, const FLootLockerServerSetMetadataResponseBP& OnCompleteBP, const FLootLockerServerSetMetadataResponseDelegate& OnComplete)
+{
+	if (SourceID.IsEmpty())
```

Comment:
Not entirely sure since I haven't tested this, but is "IsEmpty" enough here? Game feedback does not require a sourceID.

Reply:
I think so. This is metadata and not feedback so there is no "Game" level metadata to operate on. Source is an enum so it can't not have a value, and an empty action array is technically valid. So SourceID being empty is really the only thing I can check at this time. The alternative would be to check nothing I think, which I guess could be valid as well, the BE will tell the user if that is the case.